src/index.js

```diff
@@ -38,6 +39,10 @@ const router = createBrowserRouter([
         path: 'stories/button',
         element: <ButtonStory />,
       },
+      {
+        path: 'components/notification',
```

Comment:
Just use a single flow to display components.

```suggestion
        path: 'stories/notification',
```